test/models/docker_builder_service_test.rb

```diff
@@ -1,6 +1,6 @@
 require_relative '../test_helper'
 
-SingleCov.covered! uncovered: 19
+SingleCov.covered! uncovered: 20
```

Comment:
The ```run!``` method's body is uncovered and originally had 19 lines. I added an if check for running the remote job and now the body has one extra line. Do you have any idea how to test the code block passed to the job execution inside this method's body?